src/configs/latest.ts

```diff
@@ -16,6 +16,7 @@
  */
 
 export const rules = {
+    "cyclomatic-complexity": true,
```

Comment:
I am assuming in this case that I should leave it listed but marked as false? Using typedef in recommended.ts as an example.